bliss/conf/base_config.yaml

```diff
@@ -129,11 +129,10 @@ variational_factors:
 metrics:
     detection_performance:
         _target_: bliss.encoder.metrics.DetectionPerformance
-        mag_bin_cutoffs: [19, 19.4, 19.8, 20.2, 20.6, 21, 21.4, 21.8]
-        mag_band: 2
+        bin_cutoffs: [200, 400, 600, 800, 1000]
     source_type_accuracy:
         _target_: bliss.encoder.metrics.SourceTypeAccuracy
-        flux_bin_cutoffs: [200, 400, 600, 800, 1000]
+        bin_cutoffs: [200, 400, 600, 800, 1000]
```

Comment:
`bin_cutoffs` is quite vague without the word `flux`, can we avoid changing it?

Reply:
I rename it to `bin_cutoffs` because we also use `blendedness` as x-axis for some metrics.

Reply:
Oh I see. That does seem like a useful generalization then.